src/main/java/de/msg/javatraining/donationmanager/controller/campaign/CampaignController.java

```diff
@@ -28,11 +28,11 @@ public List<Campaign> getALlCampaigns(){
     @PostMapping("/{userId}") ///campaign/userId
     public ResponseEntity<?> createCapmaign(@PathVariable Long userId,@RequestBody Campaign campaign){
 
-        Campaign camp = campaignService.createCampaign(userId,campaign.getName(),campaign.getPurpose());
-        if(camp!=null){
-            return ResponseEntity.ok().build();
+        ResponseEntity<?> camp = campaignService.createCampaign(userId,campaign.getName(),campaign.getPurpose());
+        if(camp.getStatusCode() == HttpStatus.OK){
+            return ResponseEntity.ok("Campaign created successfully!");
         }else{
-            return ResponseEntity.status(HttpStatus.FORBIDDEN).build();
+            return ResponseEntity.status(HttpStatus.FORBIDDEN).body("Campaign has not been created!");
```

Comment:
A zis Sergiu ca nu ar trebui sa returnam alt status in afara de OK, pentru ca noi am interceptat eroarea si am oprit request-ul din a se face.